test/app_signals/resources/traceid_generator.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package main
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"time"
+)
+
+func main() {
```

Comment:
Isn't this generator a copy of code from a different package? Can you just use that one or move it into somewhere more common?

Reply:
This is being used in terraform template, I am not quite sure how to use the imported go file to use in terraform